tests/cypress/tests-e2e/layers.cy.js

```diff
@@ -242,6 +242,42 @@ describe('Test of layer handling', () => {
                     { id: fakeWmsLayerId3, visible: false, opacity: 1.0 },
                     { id: fakeWmsLayerId4, visible: false, opacity: 0.4 },
                 ])
+
+                cy.log('getFeatureInfo testing')
+                // layer 1 and 2 have the same "backend", so we deactivate layer 2 and activate layer 3
+                cy.get(`[data-cy="button-toggle-visibility-layer-${fakeWmsLayerUrlId2}"]`).click()
+                cy.get(`[data-cy="button-toggle-visibility-layer-${fakeWmsLayerUrlId3}"]`).click()
```

Comment:
This will be in conflict with my PR https://github.com/geoadmin/web-mapviewer/pull/726 and we need to coordinate who is merging first, probably you 😉

Reply:
Go ahead, I'll be on rebase duty this time 😄

Reply:
Ok then don't forget to add the `index` in your cypress query or use `[data-cy^="...`